src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const formattedQuery = query.toLowerCase().trim();
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const movieTitle = movie.title.toLowerCase();
+    const movieDesc = movie.description.toLowerCase();
+
+    return movieTitle.includes(formattedQuery)
+     || movieDesc.includes(formattedQuery);
+  });
```

Comment:
You can destructure the `movie` variable
```suggestion
  const visibleMovies = moviesFromServer.filter(movie => {
    const { title, description } = movie
    const movieTitle = title.toLowerCase();
    const movieDesc = description.toLowerCase();

    return movieTitle.includes(formattedQuery)
     || movieDesc.includes(formattedQuery);
  });
```